Formula/tremor-runtime.rb

```diff
@@ -0,0 +1,61 @@
+class TremorRuntime < Formula
+  desc "Early-stage event processing system for unstructured data"
+  homepage "https://www.tremor.rs/"
+  url "https://github.com/tremor-rs/tremor-runtime/archive/refs/tags/v0.12.4.tar.gz"
+  sha256 "91cbe0ca5c4adda14b8456652dfaa148df9878e09dd65ac6988bb781e3df52af"
+  license "Apache-2.0"
+  head "https://github.com/tremor-rs/tremor-runtime.git", branch: "main"
+
+  bottle do
+    root_url "https://github.com/sandipb/homebrew-tremor/releases/download/tremor-runtime-0.12.4"
+    sha256 cellar: :any_skip_relocation, monterey: "eccbc7e61188daee1e2deadf04752d5662f05ded2e68741b431a8a9523beb74a"
+    sha256 cellar: :any_skip_relocation, big_sur:  "4de4e40f728af2c1b17aef831e66a59177a8be25e93a8a8c68951d88fd27d438"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "rust" => :build
+
+  def install
+    # To fix SIGILL. https://github.com/tremor-rs/tremor-runtime/issues/1221
+    ENV["RUSTFLAGS"] = "-C target-cpu=native"
+
+    system "cargo", "install", *std_cargo_args(path: "tremor-cli")
+
+    system "sh", "-c", "target/release/tremor completions bash > bash-completion"
+    system "sh", "-c", "target/release/tremor completions zsh > zsh-completion"
+    system "sh", "-c", "target/release/tremor completions fish > fish-completion"
+    bash_completion.install "bash-completion"
+    zsh_completion.install "zsh-completion"
+    fish_completion.install "fish-completion"
+
+    # main binary
+    bin.install "target/release/tremor"
+
+    # stdlib
+    mkdir (lib/"tremor-script")
+    cp_r "tremor-script/lib/.", (lib/"tremor-script")
+
+    # sample config for service
+    mkdir (etc/"tremor")
+    cp "docker/config/docker.troy", (etc/"tremor/main.troy")
+
+    # wrapper
+    (bin/"tremor-wrapper").write <<~EOS
+      #!/bin/bash
+      export TREMOR_PATH="#{lib}/tremor-script"
+      exec #{bin}/tremor "$@"
+    EOS
+  end
+
+  service do
+    run [bin/"tremor-wrapper", "run", etc/"tremor/main.troy"]
+    keep_alive true
+    working_dir HOMEBREW_PREFIX
+    log_path var/"log/tremor.log"
+    error_log_path var/"log/tremor_error.log"
+  end
+
+  test do
+    assert_match(/#{version}/, shell_output("#{bin}/tremor --version"))
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Done